infrastructure/ai/providers/vertex_ai_provider.py

```
@@ -0,0 +1,164 @@
+            or os.getenv("GCP_REGION") 
+            or DEFAULT_REGION
+        ).strip()
+        self.project_id = (
+            settings.extra_body.get('project_id') 
+            or os.getenv("GOOGLE_CLOUD_PROJECT") 
+            or os.getenv("GCP_PROJECT_ID")
+        ).strip()
+        
+        # 2. 初始化客户端 (回归 Vertex AI 企业专线 + 注入关键计费请求头)
+        try:
+            # 使用动态读取的 project_id，确保以后改 .env 也能生效
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)